.github/workflows/release.yml

```diff
@@ -0,0 +1,43 @@
+---
+name: Release
+
+on:
+  pull_request_target:
+    types:
+      - closed
+    branches:
+      - master
+      - "[2-9][0-9].[0-9][0-9].x"
+    paths-ignore:
+      - ".github/**"
+      - ".jira/**"
+      - "tests/**"
+      - ".deepsource.toml"
+      - ".veracode-exclusions"
+      - "README.md"
+      - "sonar-project.properties"
+  pull_request:
+    paths:
+      - ".github/workflows/release.yml"
+  workflow_dispatch:
+
+jobs:
+  release:
+    # DO UNCOMMENT LINE BELOW BEFORE MERGING
```

Comment:
do uncomment? wdym?

Reply:
indeed i ll removed it, this was a hard reminder